### DISCUSSION TITLE
Support Fontawesome kits & custom icons

### DISCUSSION THREAD

**Discussion by RCVarley** [quasarframework/quasar#17397] [Ideas / Proposals]
Fontawesome offers custom kits which can contain custom icons. 

My request is for the ability to register a custom kit as an `iconSet`. 

> Note: If this is already supported, could you please update the documentation to show how? Or tell me how to do it and I'll submit an update.

FYI, when uploading a custom duotone icon called _store-group_, the basic html implementation would look like this:
`<i class="fa-kit-duotone fa-store-group"></i>`.

I don't know if anything needs to happen for QIcon to handle the `fa-kit-duotone` class.

**Comment by ajmas** [comment]
Maybe start with the following page (it already covers some customisation info), and see if it helps:

https://quasar.dev/vue-components/icon

**Comment by jacobbutterly** [comment]
I know it's been almost a year, but how did you get on with this in the end? I can't seem to get the Font Awesome Kit loaded.
I've been using `import '@awesome.me/kit-xxxxxxxxxx/icons/css/fontawesome.css'` in the new `boot/fontawesome-pro.ts` file I created and added in `quasar.conf.js`.

When I load the page, I get this error:
```
[plugin:vite:import-analysis] Missing "./icons/css/fontawesome.css" export in "@awesome.me/kit-xxxxxxxxxx" package
```

**Reply by stefanvanherwijnen** [reply]
Give [@unocss/preset-icons](https://unocss.dev/presets/icons) a try.
